### DISCUSSION TITLE
Option to disable scrolling modifiers on Apple internal keyboard / Trackpad

### DISCUSSION THREAD

**Discussion by SigurdRokenes** [linearmouse/linearmouse#477] [Q&A]
Is it possible to disable the scrolling modifiers on the Apple trackpad? Currently the system default (Accelerated, acceleration=1, scrolling speed=0) makes scrolling using the trackpad way too fast on the trackpad, and even with tweaking I can never find a setting that feels as smooth as the original scrolling. I find myself in the situation where I keep opening and closing LinearMouse depending on if I use the mouse or the keyboard. Is it possible to add a setting to disable scrolling (or all settings) for the trackpad?

**Comment by lujjjh** [comment]
Could you please attach your configuration file? You can find it by going to Config -> Reveal in Finder.

**Reply by SigurdRokenes** [reply]
Well, I found the config file and saw that the scrolling was changed for my browser only on the trackpad. I deleted this and now the default settings function great again. Issue can be closed.
